packages/assets/package.json

```diff
@@ -9,7 +9,12 @@
     "directory": "packages/assets"
   },
   "homepage": "https://github.com/facebook/react-native/tree/HEAD/packages/assets#readme",
-  "keywords": ["assets", "registry", "react-native", "support"],
+  "keywords": [
+    "assets",
+    "registry",
+    "react-native",
+    "support"
+  ],
```

Comment:
Did Yarn do this? Wouldn't this cause unnecessary diffs?

Reply:
Yep, hence: https://github.com/facebook/react-native/pull/42256
I didn't see an option in `.yarnrc.yml` to not lint on every `yarn install`... so I'm gonna hold off a bit and see if the above PR merges.

Reply:
Aaaand it merged.

Reply:
For stuff like back ports / cherry picks (which I guess this change will kind of be), I usually dont' add the macOS diff tags, since we'll get there eventually. Also you can't add comments easily to package.json files. So I'm OK merging as is.